README.md

```diff
@@ -1,24 +1,74 @@
-# README
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+# Budget App
 
-Things you may want to cover:
+A Budget-app is mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
 
-* Ruby version
+![screenshot](./screenshot-1.png)
```

Comment:
### OPTIONAL
- [ ] Kindly fix the broken link to make a professional readme in the screenshot.

Reply:
I have fixed the link for the screenshot

thank you